Core/src/main/java/dk/sdu/se4/core/Game.java

```diff
@@ -5,25 +5,35 @@
 import com.badlogic.gdx.graphics.GL20;
 
 public class Game implements ApplicationListener {
+
+    private ServicesStorage servicesStorage;
+
     @Override
-    public void create() {}
+    public void create() {
+        this.servicesStorage = new ServicesStorage();
```

Comment:
I still don't think you should have a class just for the services. I think it adds unnessesary complexity.

Reply:
I think is a good choice to give the responsibility to a specified class :) ane class shot only have small responsibility